modules/creating-machines-bare-metal.adoc

```diff
@@ -0,0 +1,32 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_bare_metal/installing-bare-metal.adoc
+// * installing/installing_bare_metal/installing-restricted-networks-bare-metal.adoc
+// * installing/installing_platform_agnostic/installing-platform-agnostic.adoc
+
+[id="creating-machines-bare-metal_{context}"]
+= Installing {op-system} and starting the {product-title} bootstrap process
```

Comment:
None of this content is net new; it's all existed for several versions as is in the docs. This PR just breaks out this as a module instead of inline content that was previously part of the parent assembly file. Therefore, no need to review extensively for style and wording; more just checking to be sure headings are rendering properly.